docs/elements/hx-tablist/index.html

```diff
@@ -0,0 +1,28 @@
+---
+title: <hx-tablist>
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+<section>
+  <h2 class="hxSectionTitle" id="attributes">Attributes</h2>
+  <dl>
+    <dt>
+      aria-orientation
```

Comment:
See note regarding `code` blocks for all in this column.

Reply:
removing `aria-orientation` -- <code>tab-side</code> and <code>tabSide</code> removed